modules/logging-loki-logcli.adoc

```diff
@@ -0,0 +1,32 @@
+// Module is included in the following assemblies:
+// 
+//
+:_content-type: REFERENCE
+[id="logging-loki-logcli-about_{context}"]
+
+= Querying Loki
+
+You can use Loki's command-line interface `logcli` to query logs.
+
+.Example Application Log Query
+[source,terminal]
+----
+$ oc extract cm/lokistack-sample-ca-bundle --to=lokistack --confirm
+$ cat lokistack/*.crt >lokistack_ca.crt
+$ logcli -o raw --bearer-token="${bearer_token}" --ca-cert="lokistack_ca.crt" --addr xxxxxx
```

Comment:
One command per code block. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output

![image](https://github.com/openshift/openshift-docs/assets/40172997/6449191e-f8b3-46cb-b54e-7e62f4bc3cba)